src/utils/index.ts

```diff
@@ -295,6 +294,41 @@ export const fetchTranscriptDetails = (allTranscripts: Transcript[], paths: stri
   };
 };
 
+export const createLanguageTreeArray = (languageTree: any) => {
```

Comment:
can you add a comment on what this does specifically i.e `create sources list by language` as it doesn't return a tree.
Or about a more specific name, e.g `deriveSourcesList`